services/galley/src/Galley/API/Internal.hs

```diff
@@ -63,7 +63,7 @@ rmUser user conn = do
   Data.eraseClients user
   where
     leaveTeams tids = for_ (result tids) $ \tid -> do
-      Data.teamMembersUnsafeForLargeTeams tid >>= uncheckedRemoveTeamMember user conn tid user
+      Data.teamMembersUnsafeForLargeTeams tid >>= uncheckedRemoveTeamMember user conn tid user . Just
```

Comment:
we should not call `teamMembersUnsafeForLargeTeams` anymore, no?

Reply:
Not part of this PR?

Reply:
ah yes, it's not used to validate the deletion, but notify others about it.